package.json

```diff
@@ -148,7 +148,7 @@
     "request": "2.84.0",
     "rimraf": "2.6.2",
     "rocambole": "0.7.0",
-    "sinon": "4.4.6",
+    "sinon": "4.5.0",
```

Comment:
The latest version is 5.0.0. The release notes don't document it yet, but it's official, and installs fine when you run:
```
yarn add --dev --exact sinon@5.0.0
```

Reply:
According to [npm](https://www.npmjs.com/package/sinon?activeTab=versions), latest is `4.5.0` and next is `5.0.1-next.1`. 5.0.0 was unpublished 3 months ago (likely an accidental publish).

Reply:
SGTM. We'll have to remove the deprecated usage in our code when 5.0.x gets published, but that can wait.

Reply:
Version 5.0.1 is now out (#14957). There are several new errors that are popping up.

https://travis-ci.org/ampproject/amphtml/jobs/373092869